Docs/source/developers/profiling.rst

```diff
@@ -43,6 +79,25 @@ Example on how to create traces on a multi-GPU system that uses the Slurm schedu
        ./warpx.3d.MPI.CUDA.DP.QED inputs_3d \
          warpx.numprocs=1 1 4 amr.n_cell=512 512 2048 max_step=10
 
+ Example on how to create traces on a multi-GPU system that uses the ``jsrun`` scheduler (e.g., `OLCF's Summit system <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#optimizing-and-profiling>`__):
+
+ .. code-block:: bash
+
+    # nsys: remove old traces
+    rm -rf profiling* tmp-traces
+    # nsys: a location where we can write temporary nsys files to
+    export TMPDIR=$PWD/tmp-traces
```

Comment:
Could we add `mkdir -p $TMPDIR` here?
When I tried this workflow, it failed at first, because that directory did not exist.

Reply:
Ah, that's good to know! Definitely!
```suggestion
    export TMPDIR=$PWD/tmp-traces
    mkdir -p $TMPDIR
```